packages/teams-js/src/private/interfaces.ts

```diff
@@ -56,108 +56,108 @@ export enum ViewerActionTypes {
 }
 
 /**
- * * @privateRemarks
+ * * @hidden
```

Comment:
There's an extraneous asterisk before @hidden (line 59) that's causing the "Hide from docs" text to generate documentation

Reply:
Great catch!